Formula/librespot.rb

```diff
@@ -0,0 +1,20 @@
+class Librespot < Formula
+  desc "Open Source Spotify client library"
+  homepage "https://github.com/librespot-org/librespot"
+  url "https://github.com/librespot-org/librespot.git",
+      tag:      "v0.2.0",
+      revision: "59683d7965480e63c581dd03082ded6a080a1cd3"
+  license "MIT"
+  head "https://github.com/librespot-org/librespot.git",
+      branch: "dev"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--no-default-features", "--features", "rodio-backend,with-dns-sd", *std_cargo_args
+  end
+
+  test do
+    system "#{bin}/librespot", "-V"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Yeah, I know, but as I wrote in the PR description:

> The test is lacking, as I haven't found a way of making librespot just try to connect and then exit. It just keeps running until you manually close it, so I don't know how to test it.

Reply:
Can't we run it for a little while, get it to dump some output we can verify, then quit?